### ISSUE TITLE
TouchPad not working after Installation / It was working when tested with LIVE USB

### ISSUE THREAD

**Issue by ghost** [geckolinux/geckolinux-project#156]
Hello All,
I reached to GeckoLinux by searching for a smaller size OpenSuse Installer ISO. I read some reviews and I wanted to install this distro. Today I created a LIVE USB from [GeckoLinux_ROLLING_Plasma.x86_64-999.200729.0.iso](https://github.com/geckolinux/geckolinux-project/releases/tag/999.200729) and tested it against my machine Lenovo ideapad 330-15ARR 
Everything worked well,  but no WIFI. My WIFI driver is **RTL8821CE 802.11ac** which is infamous for No-Support in various LinuxDistros. Out of 7 or more Distros I tried, only Mageia became successful to auto-detect it. But I dont worry since I found [this](https://build.opensuse.org/package/show/home%3ASauerland%3Ahardware/rtl8821ce), I will try it.

 But my problem is no TouchPad. ] It was working fine when I tested with LiveUSB. The touchpad was working in my Mageia 8 also.   Libinput info below 
````
jeevanism-81d2:/home/jeevanism # libinput list-devices
Device:           Power Button
Kernel:           /dev/input/event6
Group:            1
Seat:             seat0, default
Capabilities:     keyboard 
Tap-to-click:     n/a
Tap-and-drag:     n/a
Tap drag lock:    n/a
Left-handed:      n/a
Nat.scrolling:    n/a
Middle emulation: n/a
Calibration:      n/a
Scroll methods:   none
Click methods:    none
Disable-w-typing: n/a
Accel profiles:   n/a
Rotation:         n/a

Device:           Video Bus
Kernel:           /dev/input/event1
Group:            2
Seat:             seat0, default
Capabilities:     keyboard 
Tap-to-click:     n/a
Tap-and-drag:     n/a
Tap drag lock:    n/a
Left-handed:      n/a
Nat.scrolling:    n/a
Middle emulation: n/a
Calibration:      n/a
Scroll methods:   none
Click methods:    none
Disable-w-typing: n/a
Accel profiles:   n/a
Rotation:         n/a

Device:           Video Bus
Kernel:           /dev/input/event2
Group:            2
Seat:             seat0, default
Capabilities:     keyboard 
Tap-to-click:     n/a
Tap-and-drag:     n/a
Tap drag lock:    n/a
Left-handed:      n/a
Nat.scrolling:    n/a
Middle emulation: n/a
Calibration:      n/a
Scroll methods:   none
Click methods:    none
Disable-w-typing: n/a
Accel profiles:   n/a
Rotation:         n/a

Device:           Power Button
Kernel:           /dev/input/event5
Group:            3
Seat:             seat0, default
Capabilities:     keyboard 
Tap-to-click:     n/a
Tap-and-drag:     n/a
Tap drag lock:    n/a
Left-handed:      n/a
Nat.scrolling:    n/a
Middle emulation: n/a
Calibration:      n/a
Scroll methods:   none
Click methods:    none
Disable-w-typing: n/a
Accel profiles:   n/a
Rotation:         n/a

Device:           Lid Switch
Kernel:           /dev/input/event4
Group:            4
Seat:             seat0, default
Capabilities:     switch
Tap-to-click:     n/a
Tap-and-drag:     n/a
Tap drag lock:    n/a
Left-handed:      n/a
Nat.scrolling:    n/a
Middle emulation: n/a
Calibration:      n/a
Scroll methods:   none
Click methods:    none
Disable-w-typing: n/a
Accel profiles:   n/a
Rotation:         n/a

Device:           HD-Audio Generic HDMI/DP,pcm=3
Kernel:           /dev/input/event9
Group:            5
Seat:             seat0, default
Capabilities:     
Tap-to-click:     n/a
Tap-and-drag:     n/a
Tap drag lock:    n/a
Left-handed:      n/a
Nat.scrolling:    n/a
Middle emulation: n/a
Calibration:      n/a
Scroll methods:   none
Click methods:    none
Disable-w-typing: n/a
Accel profiles:   n/a
Rotation:         n/a

Device:           SIGMACHIP Usb Mouse
Kernel:           /dev/input/event3
Group:            6
Seat:             seat0, default
Capabilities:     pointer 
Tap-to-click:     n/a
Tap-and-drag:     n/a
Tap drag lock:    n/a
Left-handed:      disabled
Nat.scrolling:    disabled
Middle emulation: disabled
Calibration:      n/a
Scroll methods:   button
Click methods:    none
Disable-w-typing: n/a
Accel profiles:   flat *adaptive
Rotation:         n/a

Device:           EasyCamera: EasyCamera
Kernel:           /dev/input/event12
Group:            7
Seat:             seat0, default
Capabilities:     keyboard 
Tap-to-click:     n/a
Tap-and-drag:     n/a
Tap drag lock:    n/a
Left-handed:      n/a
Nat.scrolling:    n/a
Middle emulation: n/a
Calibration:      n/a
Scroll methods:   none
Click methods:    none
Disable-w-typing: n/a
Accel profiles:   n/a
Rotation:         n/a

Device:           HD-Audio Generic Mic
Kernel:           /dev/input/event10
Group:            5
Seat:             seat0, default
Capabilities:     
Tap-to-click:     n/a
Tap-and-drag:     n/a
Tap drag lock:    n/a
Left-handed:      n/a
Nat.scrolling:    n/a
Middle emulation: n/a
Calibration:      n/a
Scroll methods:   none
Click methods:    none
Disable-w-typing: n/a
Accel profiles:   n/a
Rotation:         n/a

Device:           HD-Audio Generic Headphone
Kernel:           /dev/input/event11
Group:            5
Seat:             seat0, default
Capabilities:     
Tap-to-click:     n/a
Tap-and-drag:     n/a
Tap drag lock:    n/a
Left-handed:      n/a
Nat.scrolling:    n/a
Middle emulation: n/a
Calibration:      n/a
Scroll methods:   none
Click methods:    none
Disable-w-typing: n/a
Accel profiles:   n/a
Rotation:         n/a

Device:           Ideapad extra buttons
Kernel:           /dev/input/event8
Group:            8
Seat:             seat0, default
Capabilities:     keyboard 
Tap-to-click:     n/a
Tap-and-drag:     n/a
Tap drag lock:    n/a
Left-handed:      n/a
Nat.scrolling:    n/a
Middle emulation: n/a
Calibration:      n/a
Scroll methods:   none
Click methods:    none
Disable-w-typing: n/a
Accel profiles:   n/a
Rotation:         n/a

Device:           AT Translated Set 2 keyboard
Kernel:           /dev/input/event0
Group:            9
Seat:             seat0, default
Capabilities:     keyboard 
Tap-to-click:     n/a
Tap-and-drag:     n/a
Tap drag lock:    n/a
Left-handed:      n/a
Nat.scrolling:    n/a
Middle emulation: n/a
Calibration:      n/a
Scroll methods:   none
Click methods:    none
Disable-w-typing: n/a
Accel profiles:   n/a
Rotation:         n/a
````

System Info :
` Kernel: 5.7.9-1-default 
  CPU: AMD Ryzen 5 2500U with Radeon Vega Mobile Gfx (8) @ 2.000GHz 
  GPU: AMD ATI Radeon Vega Series / Radeon Vega Mobile Series 
`

Any help is appreciate - thank you :+1:

**Comment by ghost** [comment]
UPDATE : 
The ToucPad started working after a Reboot :) Dont know why it shows in my LibInput list
`Device:           Synaptics TM3336-001`

thank you :)

**Comment by ghost** [comment]
Ok, sorry to bug again. I confirmed my touchpad worked after reboot but ever since then I was using my external USB mouse, so I did not check about the status of my TouchPad Mouse. Today I did a zypper dup, I am not sure whether this is the reason or  not but just found out the touchpad is not working no more. 

libinput is not showing my Synaptics. 

```
jeevanism@jeevanism-81d2:~/.kde4/share/config> uname -r
5.8.7-1-default
```
and I use the Tumbleweed version.  Is there any driver I should install ?

**Comment by geckolinux** [comment]
Hi there, the Synaptics driver is probably not installed, since most desktop environments now use libinput instead.

**Comment by ghost** [comment]
**|| Synaptics driver is probably not installed**
Is the driver available via zypper ?

**Comment by ghost** [comment]
it was working under - Kernel: 5.7.9-1-default
but after zypper dup  its 5.8.7-1-default now - this might be an issue ?

**Comment by geckolinux** [comment]
Hello, there are probably multiple kernel versions installed, which is normal. Please use `uname -a` to see which one is currently booted.

**Comment by ghost** [comment]
Update - 
I booted into **5.7.9-1-default** now, the touchPad not working here too, wireless driver not working. Wifi is working in **5.8.7-1-default**  fine. 

So I will try to fix the touchPad driver issue in **5.8.7-1-default** itself !

**Comment by ghost** [comment]
again UPDATE - the touchPad is working in 5.8.7-1-default now ! I dont know whats going on, seems like some kind of erratic activity happening.. Any how, I guess this thread gotta be closed, since the TouchPad worked atleast once. 

thank you and apart from this - the Distro is super Cool, polished and responsive..
 Good Work Gecko, thanks again
